src/python/bin/aqueduct

```diff
@@ -244,17 +265,20 @@ def ui(expose_ip):
         server_ip = "localhost"
 
     generate_env_local(server_ip)
-    execute_command(["npm", "run", "start"], os.path.join(ui_directory, "app"))
+    return execute_command_nonblocking(["npm", "run", "start"], os.path.join(ui_directory, "app"))
 
-def apiKey():
+def get_apikey():
     config_file = os.path.join(server_directory, "config", "config.yml")
     with open(config_file, "r") as f:
         try:
-            print(yaml.safe_load(f)['apiKey'])
+            return yaml.safe_load(f)['apiKey']
         except yaml.YAMLError as exc:
             print(exc)
             exit(1)
 
+def apiKey():
```

Comment:
Style nit -- can we be consistent about this throughout the code? We have `apikey` some places and `apiKey` in others.